example/burrow_integration/integ-ex.py

```diff
@@ -0,0 +1,171 @@
+import os
```

Comment:
Please rename this file more verbosely, less abbreviations. Looks like its name should be 'integration extension'